bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Extensions/ComputeCommitment.cs

```diff
@@ -61,7 +61,7 @@ public static class ComputeCommitmentExtensions
     /// Compute the commitment for the selection using 
     /// the publically available data.
     ///
-    /// equations (63) and (64) in the spec
+    /// equations (74) and (75) in the v2.0.0 spec
```

Comment:
i tried to update the numbers everywhere i saw them, and add the spec version so that at least it's easier to track this stuff down. i id not do it everywhere, only the fils i touched.